sonar-vbnet-plugin/pom.xml

```diff
@@ -50,7 +50,6 @@
     <dependency>
       <groupId>org.sonarsource.api.plugin</groupId>
       <artifactId>sonar-plugin-api</artifactId>
-      <version>${sonar.version}</version>
       <scope>provided</scope>
```

Comment:
Can we have a comment explaining what is providing the `sonar-plugin-api` (I guess SonarQube/SonarCloud) and why we need the scope defined at this level?

As for the second question: I am not sure we need it here: the scope is already defined as `provided` in the `dependencyManagement` of the pom at root-level.

Reply:
Indeed. I removed all but the one at the root level.